### PR TITLE
[DOC-522] [2024.2] Rolling operations in batches and YBA compatibility for 2024.2

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /v2024.2/yugabyte-platform/manage-deployments/edit-config-flags/#rolling-restart-in-batches

### CODE REVIEW

**Review by yorq — CHANGES_REQUESTED** [review]
Shouldn't we mention that this functionality is also available for other upgrades (but only by using runtime config)?

**Comment by yorq** [comment]
@rajmaddy89 Yes, but I think it is ok not to mention it in docs

**Review by yorq — APPROVED** [review]
LGTM

**Review by ddhodge — COMMENTED** [review]
made some changes

File: docs/content/v2024.2/yugabyte-platform/manage-deployments/edit-config-flags.md [hunk 1]
```diff
@@ -37,10 +37,22 @@ To customize flags of the read replica of a universe that has a read replica clu
...
 Depending on the flag, the universe may need to be restarted to apply the changes. You can apply changes as follows:
 
-- Immediately using a rolling restart.
+- Immediately using a rolling restart, or [rolling restart in batches](#rolling-restart-in-batches).
 - Immediately using a concurrent restart.
 - Immediately apply any changes that do not require a restart and wait until the next time the universe is restarted to apply the remaining changes.
 
+### Rolling restart in batches
+
+YugabyteDB Anywhere allows processing multiple nodes simultaneously within each availability zone during rolling operations. YBA supports this rolling batch feature which is currently {{<tags/feature/ea>}} for TServer nodes, but Master nodes are always updated one at a time.
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/v2024.2/yugabyte-platform/manage-deployments/edit-config-flags.md]: cc @yorq @rajmaddy89

[the remark this answers is not in the document]
Comment (yorq) [reply docs/content/v2024.2/yugabyte-platform/manage-deployments/edit-config-flags.md]: @ddhodge Then it depends on whether we apply upgrade on the whole node or on the process.
If the whole node (for ex ResizeNode) - we process this node one at a time
If we apply to process (GFlags upgrade) - then we process master on this node and later when all masters are processed we split tservers to batches

**Pull request merged**